### ISSUE TITLE
Plugin adds extra text to label and button elements.

### ISSUE THREAD

**Issue by stljeff1** [artf/grapesjs-plugin-forms#38]
I am experiencing odd behavior related to this plugin and am looking for insight and context.

When I drag a block containing a `label` or `button` tag, SOMETHING is inserting text into the tag. On `label` tags, "Label" gets inserted, and on  `button` tags, `Send` gets inserted.

Look at screenshot:
![screenshot-localhost_4200-2022 08 04-12_28_31](https://user-images.githubusercontent.com/9561866/182936802-31b54ead-1770-46af-8245-30014dcdb9c9.png)

i've tried to duplicating the error using jsFiddle but cannot duplicate. I am hoping someone might have insight or other feedback on why this might be happening

jsFiddle: https://jsfiddle.net/fxpmgcLu/1/

Basically, if i do not include this plugin (in my instance, everything is fine and dandy. Otherwise, my label and button tags get this extra text. I suspect this "bug" is more my fault rather than a bug in the plugin. Still, it is odd that everything is ok when I take out this plug

**Comment by artf** [comment]
Hi @stljeff1 that looks like an old issue with the previous version of this plugin, ensure to use the latest version.

**Comment by albertoVieiraNeto** [comment]
Hi, this is still happening on the version 2.0.5 available as the "latest" on npmjs
